### ISSUE TITLE
Balance tranforms to Unconfirmed Transaction - GeorgeYAN

### ISSUE THREAD

**Issue by GerogeYan** [bitcoin-solutions/multibit-hd#874]
Hi I have an issue of my multibit hd wallet. M wallet balance transformed itself to an Unconfirmed Transaction. I tried to repair my wallet , it will solve the problem at the begaining. But it doesnt work now. All transaction shows unconfirmed , and lots of finished transactions shows in my send/receive list ( you can see the attachment) I have deleted the temp file. and restored wallet. even restore it on a new workstation. It still doesnt work. Could you help to check on this, Thank you very much!
(BY THE WAY. All the transactions in my wallet have been confirmed . I checked on BlockChain.info)
![1](https://cloud.githubusercontent.com/assets/17687189/13554794/5611154e-e3ec-11e5-9e9c-c95ab182beef.jpg)
![2](https://cloud.githubusercontent.com/assets/17687189/13554795/56116d0a-e3ec-11e5-8ac2-fbdd15615754.jpg)

**Comment by jim618** [comment]
Have a read of:
https://github.com/bitcoin-solutions/multibit-hd/issues/862

which your post is a duplicate of.

**Comment by GerogeYan** [comment]
Yes I followed the #862 who has the same issue step by step. But I still can not fix my wallet

**Comment by GerogeYan** [comment]
@jim618 and I open the detail of the transaction , the status is "Transaction status unknown"

**Comment by jim618** [comment]
Probably best if you do a repair wallet and - whilst everything looks good - then do a Tools | Empty wallet to some other wallet.
There are probabilistic fixes that have gone into the code but you probably don't want to wait until that's released.

**Comment by GerogeYan** [comment]
Hi Jim

 

I do really want to empty my wallet to another address. But the balance is uncomfirmed that make me can not do any transaction now. 

Is there any other way to send it to other wallet ?

 

Thanks

 

发件人: Jim Burton [mailto:notifications@github.com] 
发送时间: 2016年3月6日 22:56
收件人: bitcoin-solutions/multibit-hd <multibit-hd@noreply.github.com>
抄送: GerogeYan <george.yan@bitse.com>
主题: Re: [multibit-hd] Balance tranforms to Unconfirmed Transaction - GeorgeYAN (#874)

 

Probably best if you do a repair wallet and - whilst everything looks good - then do a Tools | Empty wallet to some other wallet.
There are probabilistic fixes that have gone into the code but you probably don't want to wait until that's released.

**Comment by jim618** [comment]
1. Rename wallet dir as per the bitcointalk notes
2. Restore from wallet words
3. Whilst wallet looks good, do Tools | Empty wallet

**Comment by jim618** [comment]
Alternatively:
1. Download Lighthouse (http://www.vinumeris.com/lighthouse)
2. Restore form wallet words
3, Empty wallet in Lighthouse

**Comment by GerogeYan** [comment]
@jim618 Thank you very much for your help . About the Lighthouse. The link of download is not available. Is there any tool like this ?

**Comment by jim618** [comment]
You can also use Breadwallet on iOS which uses the same key derivation.
There isn't a wallet on Android that uses the same key derivation unfortunately.

**Comment by jim618** [comment]
I just downloaded the Mac version of Lighthouse fine.
You have to click on the link text itself rather than the OS image.

**Comment by qeioz** [comment]
@jim618 I’ve tried everything in last few days and still nothing worked. I have a lot of money blocked by this bug.
I’ve been using this wallet for very long time, so it has long transaction history.
Things I’ve tried:
1. wallet repair (over 20 times)
2. wallet restoration on same PC and on another PC with freshly installed differnet OS.
3. breadwallet ONLY supports 12 words key and mine is longer, so I can’t use it as well as lightroom project seems discontinued (info on github+links don’t work)
4. tried restoration using older version on another PC ( 0.3.0 after 0.4.1 on main )
5. tried on both mac and windows

During restoration process about 3% of transactions have status «Unknown transaction status» while having over 40k confirmations on blackchain, but at some point (60% progress last try) they ALL turn to same unknown status and new ones seem ok until next point and so on. By the end it’s always my whole balance is unconfirmed.
Please, let me know if there is any hope to get my money restored. Is there any alternative to breadwallet that supports over 12 words long keys or any help at all would be great.

**Comment by qeioz** [comment]
I finally got it solved after 3 days of trying all suggestions I was able to find by installing 4th copy of windows on new VM and old version 0.1.0 of multibit HD. Restored wallet from seed (3-4 hours) and this time it worked for some reason.

**Comment by ghost** [comment]
Hey, consider putting a red banner with the following warning in the site:

"This wallet contains a serious bug reported 1 year ago that can make you balance locked and effectively unusable. A couple of lengthy workarounds exist but may not always work. Use at your own risk"

**Comment by 7genius** [comment]
gustavonalle: Good point. 
Knowing this I would use different wallet. It is very strange when I day all balance is ok and after other login one part change status to recieving and uncomfirmed balance.
In my case 1x repair and all ok. So maybe more transaction more serious problem.

**Comment by jornj** [comment]
I've decided to dig a bit deeper into why my bitcoins are 'locked'.
With Multibit HD - version 0.4.1:

All my transactions are in an unknown state, setting my wallet as unconfirmed.

What I've tried:
* Renamed the wallet directory and restored from wallet words with no timestamp.
* Repairing the wallet.

No change - all transactions are still in an unknown state.

This is one of the transactions that are in the unknown state:
https://blockchain.info/tx-index/4933c99b2f5cd980439e619c740ac3a75559a1931f7bec72714a554a04f374b3

By looking at this transaction with the details view in MBHD - The status is listed as "Unknown transaction status". 

Any workarounds I've not tried?

**Comment by lolo05** [comment]
@jornj when you say "renamed the wallet directory" what you do is rename the folder where the software is installed? This is, if the folder is "c:/program files/multi bit hd" what I should do is renaming to "c:/program files/multi bit hd restore", for example?

I have read this solution before, but I am not sure what really means.

Thanks for any help! I found this problem in my wallet today :(

**Comment by tyrex25** [comment]
Guys I think you can get into your wallet via this web wallet https://coin.space/ using your 12 words. I tried it and it was working, could see history of transactions of my wallet, but I didn't try transfer, as I already solved my problem few weeks ago by another way (using old version of wallet 0.1.0).

**Comment by jornj** [comment]
I installed an old version of Multibit HD, I believe it was 0.1.1. Restored the wallet and transferred the cash to Coinbase. Then found another app, BreadWallet, that also supported the same 'magic words'. I could basically have restored the wallet with that app instead.
So, unfortunately, no longer a Multibit HD user.
